src/utils/metrics.ts

```diff
@@ -0,0 +1,94 @@
+import client from "prom-client";
+
+export class MetricsService {
+  static apiRequestCounter = new client.Counter({
+    name: "api_request_count",
+    help: "Total number of API requests",
+    labelNames: ["method", "path", "status"],
+  });
+  static blockWatcherBlockHeight = new client.Gauge({
+    name: "block_watcher_block_height",
+    help: "Block height of the block watcher",
+    labelNames: ["chain_id"],
+  });
+  static blockWatcherReorgCount = new client.Counter({
+    name: "block_watcher_reorg_count",
+    help: "Total number of reorgs",
+    labelNames: ["chain_id"],
+  });
+  static blockWatcherBlockTime = new client.Gauge({
+    name: "block_watcher_block_time",
+    help: "Time since last block",
+    labelNames: ["chain_id"],
+  });
+  static blockWatcherNumEventsProcessed = new client.Counter({
+    name: "block_watcher_num_events_processed",
+    help: "Total number of events processed",
+    labelNames: ["chain_id"],
+  });
+  static newOwnerCount = new client.Counter({
+    name: "add_contract_new_contract_count",
+    help: "Total number of new contracts added",
+    labelNames: ["chain_id"],
+  });
+  static addContractSingleOrderCount = new client.Counter({
+    name: "add_contract_single_order_count",
+    help: "Total number of single orders added",
+    labelNames: ["chain_id"],
+  });
+  static addContractMerkleRootSetCount = new client.Counter({
+    name: "add_contract_merkle_root_set_count",
+    help: "Total number of merkle root sets added",
+    labelNames: ["chain_id"],
+  });
+  static addNewOwnerErrorCount = new client.Counter({
+    name: "add_contract_error_count",
+    help: "Total number of errors adding contracts",
+    labelNames: ["chain_id", "function"],
+  });
+  static addContractRunDuration = new client.Histogram({
+    name: "add_contract_run_duration",
```

Comment:
You need to add the units `_duration_second` as per https://prometheus.io/docs/practices/naming/

Reply:
Have done so. Per the naming practices as well, have used the plural form of the `seconds`.